crates/starknet-server/tests/common/mod.rs

```diff
@@ -1,53 +1,78 @@
-use std::net::TcpListener;
-use std::process::{Child, Command, Stdio};
-use std::{thread, time};
-
-use hyper::client::HttpConnector;
-use hyper::{Client, StatusCode, Uri};
-use thiserror::Error;
-
-#[derive(Error, Debug)]
-pub enum TestError {
-    #[error("Cannot start Devnet")]
-    DevnetNotStartable,
-    #[error("No free ports")]
-    NoFreePorts,
+#[cfg(test)]
+pub mod constants {
+    pub const HOST: &str = "localhost";
+    pub const MIN_PORT: u16 = 1025;
+    pub const MAX_PORT: u16 = 65_535;
+    pub const SEED: usize = 42;
+    pub const ACCOUNTS: usize = 3;
+
+    pub const MAX_RETRIES: usize = 10;
+
+    // predeployed account info with seed=42
+    pub const PREDEPLOYED_ACCOUNT_ADDRESS: &str =
+        "0x34ba56f92265f0868c57d3fe72ecab144fc96f97954bbbc4252cef8e8a979ba";
```

Comment:
Maybe add/use constants from this file https://github.com/0xSpaceShard/starknet-devnet-rs/blob/main/crates/starknet/src/constants.rs ?

Reply:
Regarding the predeployed account address, this is the only place where it is defined, since it's specific for seed 42 (as was used in Pythonic Devnet)

Reply:
Addressed the account class hash constant importing in https://github.com/0xSpaceShard/starknet-devnet-rs/pull/64/commits/1266db9b1c4f60115b22f73e4554a93f4adc5da2